frontend/src/views/components/Legend/LegendTimeline/LegendTimeline.component.tsx

```diff
@@ -0,0 +1,43 @@
+import Datepicker from 'views/shared/datepicker/component';
```

Comment:
Import the index file instead:
```suggestion
import Datepicker from 'views/shared/datepicker';
```